src/asm2wasm.h

```diff
@@ -879,6 +878,76 @@ class Asm2WasmBuilder {
     return ret;
   }
 
+  Expression* makeTrappingFloatToInt64(bool signed_, Expression* value) {
```

Comment:
Yeah, I considered this, but the issue is that the js-handling part is very different (since i64s don't exist in js). Given that, I couldn't find a nice way to write this, without having a bunch of extra weird params.